### PR TITLE
add hastic

### PR DESCRIPTION
**Pull request by jonyrock**
### Application name / category
[hastic](https://hastic.io) / Analytics

### Source URL
https://github.com/hastic

### Why it is awesome
You need analyse your timeseries but you can`t apply machine learning algorithms to your fat 
and fast awesome timeseries datasources and monitoring tools mentioned here without using not-awesome software. You can hack analytics for your very specific case on Python.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,6 +106,7 @@ _Web Analytics_
 - [Countly](https://count.ly) - Real time mobile and web analytics, crash reporting and push notifications platform. ([Source Code](https://github.com/countly)) `AGPL-3.0` `Javascript`
 - [Druid](http://druid.io/) - Distributed, column-oriented, real-time analytics data store. ([Source Code](https://github.com/druid-io/druid/)) `Apache-2.0` `Java`
 - [GoAccess](http://goaccess.io/) - Real-time web log analyzer and interactive viewer that runs in a terminal. ([Source Code](https://github.com/allinurl/goaccess)) `GPL-2.0` `C`
+- [Hastic](https://hastic.io) - Hackable time series pattern recognition tool with UI for Grafana ([Source Code](https://github.com/hastic)) `Apache-2.0` `Python` `Nodejs`
```
Comment (Kickball) [inline README.md]: Please can you end the description with a full stop?

Please can you merge the language tags and separate them with a "/" (without a space)?

**Comment by jonyrock** [comment]
@Kickball My bad. I fixed.

**Comment by Kickball** [comment]
Thanks for making the change.

Is TypeScript used for the backend side of the software? Otherwise, it shouldn't be listed.

I'm not 100% sure how TypeScript and Nodejs overlap, so let me know if you think it should stay (along with why).

**Comment by jonyrock** [comment]
@Kickball TypeScript used with node.js. TypeScript compiles to javascript and then runs is a regular node.js app. 

image

I don't think that TypeScript is important. I will remove it.

**Comment by jonyrock** [comment]
Hey guys @Kickball @nodiscc 
should I fix anything else? I can't sleep without this merge :)

**Comment by Kickball** [comment]
Sorry, I didn't get back to you to merge this PR, @nodiscc has sorted it though.

Thanks for making this contribution!

**Comment by jonyrock** [comment]
@Kickball @nodiscc,  thank you for this. We will make tools awesome again 😺

**Pull request merged**